.github/ISSUE_TEMPLATE/release.md

```diff
@@ -23,13 +30,25 @@ labels: maintenance
   npm publish jupyrdf-jupyter-forcegraph-$VERSION.tgz
   npm logout
   ```
+  - [ ] _URL on npm here_
+  - [ ] _URL on pypi here_
 - [ ] postmortem
-  - [ ] handle `conda-forge` feedstock tasks
+  - [ ] handle `conda-forge` [feedstock] tasks
+    - [ ] _URL on `conda-forge/ipyforcegraph-feedstock` here_
+    - [ ] _URL on `anaconda.org`_
   - [ ] validate on binder via simplest-possible gists
-    - [ ] pip
-    - [ ] conda
-  - [ ] bump to next development version
-  - [ ] bump the `CACHE_EPOCH`
-  - [ ] rebuild `yarn.lock`
-  - [ ] remove `.github/locks` and `doit lock`
-  - [ ] update release procedures with lessons learned
+    - [ ] pip `requirements.txt`
+      - [ ] _URL of pip binder here_
+    - [ ] conda `environment.yml`
+      - [ ] _URL of conda binder here_
+  - [ ] create postmortem PR
+    - [ ] _PR# here_
+    - [ ] bump to next development version
+    - [ ] bump the `CACHE_EPOCH`
+    - [ ] rebuild `yarn.lock`
+    - [ ] remove `.github/locks` and `doit lock`
```

Comment:
```suggestion
    - [ ] remove `.github/locks` and run `doit lock`
```

Reply:
5c78771, and expanded for the "least invasive" way to do it